lib/resources/r4/charge_item.yaml

```diff
@@ -0,0 +1,69 @@
+---
+name: Chargeitem
+field_name_base_url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem
+fields:
+
+  - name: code
+    required: 'Yes'
+    type: CodeableConcept
+    description: A code that identifies the charge, like a billing code.
+    action: terminology
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.code
+    binding:
+      description: A code that identifies the charge, like a billing code.
+      terminology:
+      - display: Bill Codes Types
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/CodeSystem/BillCodes
```

Comment:
Should we have a place holder for the bill code type at the end of the system?

Reply:
yes. Added [here](https://github.com/cerner/fhir.cerner.com/pull/605/commits/050ae5f567c5bc377157d50c3e1f7bcc228fd7e0#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR15).